tests/sentry/integrations/slack/notifications/test_issue_alert.py

```diff
@@ -180,6 +193,52 @@ def test_performance_issue_alert_user_block(self, occurrence):
             alert_type="alerts",
             issue_link_extra_params=f"&alert_rule_id={self.rule.id}&alert_type=issue",
         )
+        assert "level" not in blocks[3]["text"]["text"]
+        assert "release" in blocks[3]["text"]["text"]
+
+    @mock.patch("sentry.integrations.slack.message_builder.issues.get_tags", new=fake_get_tags)
+    @responses.activate
+    @with_feature("organizations:slack-block-kit")
+    def test_crons_issue_alert_user_block(self):
```

Comment:
The tests cover non regression type performance issues, and a non-error and non-performance issue category. I couldn't find any tests that created either of the regression performance issues.